corehq/apps/domain/views.py

```diff
@@ -304,6 +304,7 @@ class BaseAdminProjectSettingsView(BaseProjectSettingsView):
         access.
     """
 
+    @method_decorator(login_and_domain_required)
     @method_decorator(domain_admin_required)
```

Comment:
Good point, it does seem like domain_admin_required should imply login_and_domain_required. Will take a look.